### ISSUE TITLE
react-onsenui is unusable

### ISSUE THREAD

**Issue by steodor** [OnsenUI/OnsenUI#2034]
__Browser__ 

```
[Core]
    onsenui 2.3.1

[Framework]
    react 15.5.4

[Framework binding]
    react-onsenui 1.3.3

[Additional libraries]
    none

[Platform]
    Desktop - Windows 10

[Browser]
    Irelevant
```

__Encountered problem__

`import Ons from 'react-onsenui';` (or `require`, or importing bits like `import { Page } from 'react-onsenui';`) fails with the following error:

```
Error: Cannot find module './factoryWithTypeCheckers' from '[project-path]\node_modules\react-onsenui\dist'
    at [project-path]\node_modules\browser-resolve\node_modules\resolve\lib\async.js:55:21
    at load ([project-path]\node_modules\browser-resolve\node_modules\resolve\lib\async.js:69:43)
    at onex ([project-path]\node_modules\browser-resolve\node_modules\resolve\lib\async.js:92:31)
    at [project-path]\node_modules\browser-resolve\node_modules\resolve\lib\async.js:22:47
    at FSReqWrap.oncomplete (fs.js:114:15)
```

__How to reproduce__

I created a simple repository to outline this issue - https://github.com/steodor/onsenui-bug, but basically:
1. create a blank project
1. install `react`, `react-dom`, `onsenui`, `react-onsenui`
1. install browserify/babelify and make sure minimal react code is built and works ok
1. add `import ons from 'onsenui';` - build still works
1. add `import Ons from 'react-onsenui';` - build fails

The problem seems to be with facebook's `prop-types` plugin which now fails. Not sure what a fix would look like, so please investigate and fix asap. This issue basically renders this entire project unusable.

Thank you for your time.

**Comment by asial-matagawa** [comment]
Hi, @steodor !
It seems that webpack 2 or Babel failed to handle `require` in the following unreachable code in `facebook/prop-types`.
```js
if ("development" !== 'production') {
  var REACT_ELEMENT_TYPE = (typeof Symbol === 'function' &&
    Symbol.for &&
    Symbol.for('react.element')) ||
    0xeac7;

  var isValidElement = function(object) {
    return typeof object === 'object' &&
      object !== null &&
      object.$$typeof === REACT_ELEMENT_TYPE;
  };

  // By explicitly using `prop-types` you are opting into new development behavior.
  // http://fb.me/prop-types-in-prod
  var throwOnDirectAccess = true;
  module.exports = require('./factoryWithTypeCheckers')(isValidElement, throwOnDirectAccess);
}
```
https://unpkg.com/prop-types@15.5.10/prop-types.js

We will fix this. Thank you for reporting!

**Comment by steodor** [comment]
Great, thank you!

**Comment by asial-matagawa** [comment]
@steodor 
This issue has been fixed in #2035 and I just released `react-onsenui@1.3.4` with the fix.

Thank you for reporting!

**Comment by steodor** [comment]
Thanks!
